common/test/acceptance/pages/studio/video/video.py

```diff
@@ -142,6 +142,7 @@ def click_button_subtitles(self):
         """
         element = self.q(css='.setting-replace')[0]
         ActionChains(self.browser).move_to_element(element).click(element).perform()
+        self.wait_for_ajax()
```

Comment:
We generally try to avoid wait_for_ajax and prefer to wait for the specific thing that you are looking for to appear.

Can you click the subtitles button, then wait for the subtitles to appear? Or is there no new element or class added in this operation?